e2e/tests/start_call.spec.ts

```diff
@@ -77,6 +77,7 @@ test.describe('start new call', () => {
         await page.locator('#post_textbox').fill('/call leave');
         await page.getByTestId('SendMessageButton').click();
         await expect(page.locator('#calls-widget')).toBeHidden();
+        await expect(page.locator('#postCreateFooter .has-error')).toBeHidden();
```

Comment:
nit: or not be visible? (because it's not really hidden...) 🤷

Reply:
@cpoile Playwright semantic is the same though. It's just a bit more compact to use `toBeHidden()` vs `not.toBeVisible()`.

Otherwise we'd have to use something like `toBeAttached` which specifically checks for presence in the DOM rather than visibility.